src/components/panel/panel.js

```diff
@@ -329,21 +332,25 @@ MdPanelService.prototype.create = function(opt_config) {
  * A reference to a created panel. This reference contains a unique id for the panel,
  * along with properties/functions used to control the panel.
  *
- * @param {!angular.$injector} $injector
  * @param {!Object} config
+ * @param {!angular.$injector} $injector
  * @final @constructor
  */
-function MdPanelReference(config,$q) {
+function MdPanelReference(config, $injector) {
```

Comment:
I initially had it as MdPanelRef. @ThomasBurleson changed it to MdPanelReference. I'm okay with either answer. Thomas?